libsplinter/src/admin/service/error.rs

```diff
@@ -161,6 +161,10 @@ pub enum AdminSharedError {
         context: String,
         source: Option<InitializeServiceError>,
     },
+    ServiceShutdownFailed {
```

Comment:
This enum should be reduced to have the types defined in the `crate::error` module.  Another note to make for a future PR